src/zeek.bif

```diff
@@ -5766,3 +5766,67 @@ function have_spicy_analyzers%(%) : bool
 	%{
 	return zeek::val_mgr->Bool(USE_SPICY_ANALYZERS);
 	%}
+
+## Converts a set to a vector containing the same type as the index of the set.
+## Emits an error and returns false if the set has more than one index type, or
+## if anything other than a set is paased as an argument.
+function set_to_vector%(s: any%): any
```

Comment:
I think that `any` returns here and below are problematic because they break static typing. I realize that that's (a) because we can't know the type of the vector/set, and (b) we can't return errors otherwise. Maybe that means these functions can't/shouldn't really be BIFs? I just scanned over `zeek.bif` and it looks like we don't have any other `*_to_*` returning `any`.